guides/common/modules/con_managing-dns-using-smart-proxies.adoc

```diff
@@ -0,0 +1,23 @@
+[id="Managing_DNS_Using_Smart_Proxies_{context}"]
+= Managing DNS Using {SmartProxies}
+
+Your {SmartProxy} can manage DNS records.
+DNS management contains updating and removing DNS records from existing DNS zones.
+
+After you have enabled DNS, your {SmartProxy} can manipulate any DNS server that complies with RFC 2136.
+You can use it to manage both Microsoft Active Directory and BIND servers.
```

Comment:
This specifically talks about `dns_nsupdate` and `dns_nsupdate_gss`. Would it make sense to move it there?

Reply:
moved.